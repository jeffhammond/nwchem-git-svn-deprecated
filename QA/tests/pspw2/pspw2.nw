echo
title "GGA testing - fails on Bassi"

start ch3cl.pspw

charge 0

geometry
C             1.141270   -0.000034    0.000004
Cl           -0.664540    0.000021   -0.000006
H             1.483169   -0.877342   -0.547984
H             1.483210    0.913220   -0.485690
H             1.483190   -0.036019    1.033759
end

set nwpw:cutoff_boot_psi .false.
set nwpw:minimizer 2
############################################
############## PBE96 testing ###############
############################################

nwpw 
   simulation_cell
     ngrid 32 32 32
   end
   xc pbe96
   lcao #old default
end
task pspw energy



############################################
####### steepest descent testing ###########
############################################
nwpw
  steepest_descent
    fake_mass 400000.0d0
    time_step 15.8d0
    loop 10 10
    xc pbe96
   end
end
task pspw steepest_descent


