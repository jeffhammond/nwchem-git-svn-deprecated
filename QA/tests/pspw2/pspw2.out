 argument  1 = ../pspw2.nw



============================== echo of input deck ==============================
echo
title "GGA testing - fails on Bassi"

start ch3cl.pspw

charge 0

geometry
C             1.141270   -0.000034    0.000004
Cl           -0.664540    0.000021   -0.000006
H             1.483169   -0.877342   -0.547984
H             1.483210    0.913220   -0.485690
H             1.483190   -0.036019    1.033759
end

set nwpw:cutoff_boot_psi .false.
set nwpw:minimizer 2
############################################
############## PBE96 testing ###############
############################################

nwpw
   simulation_cell
     ngrid 32 32 32
   end
   xc pbe96
   lcao #old default
end
task pspw energy



############################################
####### steepest descent testing ###########
############################################
nwpw
  steepest_descent
    fake_mass 400000.0d0
    time_step 15.8d0
    loop 10 10
    xc pbe96
   end
end
task pspw steepest_descent


================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.5
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2013
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = WE27972
    program         = nwchem
    date            = Wed Sep 30 13:18:05 2015

    compiled        = Wed_Sep_30_12:07:06_2015
    source          = /Users/bylaska/nwchem-releases/nwchem
    nwchem branch   = Development
    nwchem revision = 27276
    ga revision     = 10576
    input           = ../pspw2.nw
    prefix          = ch3cl.pspw.
    data base       = ./ch3cl.pspw.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107196 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                           GGA testing - fails on Bassi
                           ----------------------------

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 C3  symmetry detected

          ------
          auto-z
          ------
  Looking for out-of-plane bends


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 C                    6.0000    -0.00000000     0.00000000    -1.14126965
    2 Cl                  17.0000    -0.00000000     0.00000000     0.66454035
    3 H                    1.0000    -0.58835670    -0.85075363    -1.48318932
    4 H                    1.0000    -0.44259590     0.93490866    -1.48318932
    5 H                    1.0000     1.03095260    -0.08415503    -1.48318932

      Atomic Mass 
      ----------- 

      C                 12.000000
      Cl                34.968850
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)      50.8408707965

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
    -0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             C3        
 Group number              5
 Group order               3
 No. of unique centers     3

      Symmetry unique atoms

     1    2    3



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       1.80581
    2 Stretch                  1     3                       1.08943
    3 Stretch                  1     4                       1.08943
    4 Stretch                  1     5                       1.08943
    5 Bend                     2     1     3               108.29154
    6 Bend                     2     1     4               108.29154
    7 Bend                     2     1     5               108.29154
    8 Bend                     3     1     4               110.62470
    9 Bend                     3     1     5               110.62470
   10 Bend                     4     1     5               110.62470


            XYZ format geometry
            -------------------
     5
 geometry
 C                    -0.00000000     0.00000000    -1.14126965
 Cl                   -0.00000000     0.00000000     0.66454035
 H                    -0.58835670    -0.85075363    -1.48318932
 H                    -0.44259590     0.93490866    -1.48318932
 H                     1.03095260    -0.08415503    -1.48318932

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 Cl               |   1 C                |     3.41249  |     1.80581
    3 H                |   1 C                |     2.05872  |     1.08943
    4 H                |   1 C                |     2.05872  |     1.08943
    5 H                |   1 C                |     2.05872  |     1.08943
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          4
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 Cl               |   1 C                |   3 H                |   108.29
    2 Cl               |   1 C                |   4 H                |   108.29
    2 Cl               |   1 C                |   5 H                |   108.29
    3 H                |   1 C                |   4 H                |   110.62
    3 H                |   1 C                |   5 H                |   110.62
    4 H                |   1 C                |   5 H                |   110.62
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          6
 ==============================================================================



          ****************************************************
          *                                                  *
          *               NWPW PSPW Calculation              *
          *                                                  *
          * [ (Grassmann/Stiefel manifold implementation)  ] *
          *                                                  *
          *      [ NorthWest Chemistry implementation ]      *
          *                                                  *
          *            version #5.10   06/12/02              *
          *                                                  *
          *    This code was developed by Eric J. Bylaska,   *
          *    and was based upon algorithms and code        *
          *    developed by the group of Prof. John H. Weare *
          *                                                  *
          ****************************************************
     >>>  JOB STARTED       AT Wed Sep 30 13:18:05 2015  <<<
          ================ input data ========================
  library name resolved from: compiled reference
  NWCHEM_NWPW_LIBRARY set to: </Users/bylaska/nwchem-releases/nwchem/src/nwpw/libraryps/>
 Generating 1d pseudopotential for C   

 Generated formatted_filename: ./C.vpp
  library name resolved from: compiled reference
  NWCHEM_NWPW_LIBRARY set to: </Users/bylaska/nwchem-releases/nwchem/src/nwpw/libraryps/>
 Generating 1d pseudopotential for Cl  

 Generated formatted_filename: ./Cl.vpp
  library name resolved from: compiled reference
  NWCHEM_NWPW_LIBRARY set to: </Users/bylaska/nwchem-releases/nwchem/src/nwpw/libraryps/>
 Generating 1d pseudopotential for H   

 Generated formatted_filename: ./H.vpp
  library name resolved from: compiled reference
  NWCHEM_NWPW_LIBRARY set to: </Users/bylaska/nwchem-releases/nwchem/src/nwpw/libraryps/>

 Generated formatted atomic orbitals, filename: ./C.aorb
  library name resolved from: compiled reference
  NWCHEM_NWPW_LIBRARY set to: </Users/bylaska/nwchem-releases/nwchem/src/nwpw/libraryps/>

 Generated formatted atomic orbitals, filename: ./Cl.aorb
  library name resolved from: compiled reference
  NWCHEM_NWPW_LIBRARY set to: </Users/bylaska/nwchem-releases/nwchem/src/nwpw/libraryps/>

 Generated formatted atomic orbitals, filename: ./H.aorb

 lcao guess, initial psi:ch3cl.pspw.movecs                                 
 -   spin, nalpha, nbeta:                    1                    7                    0

 input psi filename:./ch3cl.pspw.movecs

 number of processors used:         1
 processor grid           :   1 x   1
 parallel mapping         :2d hilbert
 parallel mapping         :  balanced
 number of threads        :         1
 parallel io              :       off

 options:
      boundary conditions  = periodic    (version3)
      electron spin        = restricted
      exchange-correlation = PBE96 (White and Bird) parameterization

 elements involved in the cluster:
      1: C   valence charge:  4.0000  lmax=  2
             comment    : Parameterized (J.Phys.Chem., vol 100, page 6966) Hamman psp
             pseudpotential type            :   0
             highest angular component      :   2
             local potential used           :   2
             number of non-local projections:   4
             cutoff =   0.800   0.850   0.850

      2: Cl  valence charge:  7.0000  lmax=  2
             comment    : Troullier-Martins pseudopotential
             pseudpotential type            :   0
             highest angular component      :   2
             local potential used           :   2
             number of non-local projections:   4
             semicore corrections included  :  1.300 (radius)  0.269 (charge)
             cutoff =   1.340   1.450   1.450

      3: H   valence charge:  1.0000  lmax=  1
             comment    : Parameterized (Chem.Phys.Lett., vol 322, page 447) Hamman psp
             pseudpotential type            :   0
             highest angular component      :   1
             local potential used           :   1
             number of non-local projections:   1
             cutoff =   0.800   0.800


 total charge:   0.000

 atomic composition:
     C   :    1     Cl  :    1     H   :    3

 number of electrons: spin up=     7 (   7 per task)  down=     7 (   7 per task) (Fourier space)
 number of orbitals : spin up=     7 (   7 per task)  down=     7 (   7 per task) (Fourier space)

 supercell:
      cell_name:  cell_default                                      
      lattice:    a1=<  24.000   0.000   0.000 >
                  a2=<   0.000  24.000   0.000 >
                  a3=<   0.000   0.000  24.000 >
      reciprocal: b1=<   0.262   0.000   0.000 >
                  b2=<   0.000   0.262   0.000 >
                  b3=<   0.000   0.000   0.262 >
      lattice:    a=      24.000 b=     24.000 c=      24.000
                  alpha=  90.000 beta=  90.000 gamma=  90.000
                  omega=     13824.0

      density cutoff=  8.773  fft= 32x 32x 32(     8536 waves     8536 per task)
      wavefnc cutoff=  8.773  fft= 32x 32x 32(     8536 waves     8536 per task)
      Ewald summation: cut radius=    7.64  and  1
                       Madelung Wigner-Seitz= 1.76011888 (alpha= 2.83729748 rs=14.88841178)

 technical parameters:
      time step=      5.80     ficticious mass=  400000.0
      tolerance=0.100E-06 (energy)   0.100E-06 (density)
      maximum iterations =      1000 (   10 inner    100 outer )





== Energy Calculation ==


          ============ Grassmann lmbfgs iteration ============
     >>>  ITERATION STARTED AT Wed Sep 30 13:18:09 2015  <<<
    iter.           Energy         DeltaE       DeltaRho 
    ------------------------------------------------------
     -  15 steepest descent iterations performed
      10   -0.2256221404E+02   -0.97684E-04    0.24359E-01
      20   -0.2256232173E+02   -0.18850E-06    0.68140E-06
      30   -0.2256232182E+02   -0.96411E-07    0.54877E-10
  *** tolerance ok. iteration terminated
     >>>  ITERATION ENDED   AT Wed Sep 30 13:18:12 2015  <<<


==  Summary Of Results  ==

 number of electrons: spin up=    7.00000  down=    7.00000 (real space)

 total     energy    :  -0.2256232182E+02 (   -0.45125E+01/ion)
 total orbital energy:  -0.6225367639E+01 (   -0.88934E+00/electron)
 hartree   energy    :   0.2602381543E+02 (    0.37177E+01/electron)
 exc-corr  energy    :  -0.5871075855E+01 (   -0.83873E+00/electron)
 ion-ion   energy    :   0.8125323521E+01 (    0.16251E+01/ion)

 kinetic (planewave) :   0.1088217360E+02 (    0.15546E+01/electron)
 V_local (planewave) :  -0.6516631605E+02 (   -0.93095E+01/electron)
 V_nl    (planewave) :   0.3443757529E+01 (    0.49197E+00/electron)
 V_Coul  (planewave) :   0.5204763086E+02 (    0.74354E+01/electron)
 V_xc.   (planewave) :  -0.7432613580E+01 (   -0.10618E+01/electron)
 Virial Coefficient  :  -0.1572070238E+01

 orbital energies:
    -0.2507713E+00 (  -6.824eV)
    -0.2508129E+00 (  -6.825eV)
    -0.3900610E+00 ( -10.614eV)
    -0.4039689E+00 ( -10.993eV)
    -0.4047845E+00 ( -11.015eV)
    -0.6321317E+00 ( -17.201eV)
    -0.7801535E+00 ( -21.229eV)

 Total PSPW energy     :  -0.2256232182E+02


=== Spin Contamination ===

 <Sexact^2> =    0.0000000000000000     
 <S^2>      =    0.0000000000000000     



== Center of Charge ==

spin up     (   -0.0004,    0.0000,   -0.5254 )
spin down   (   -0.0004,    0.0000,   -0.5254 )
     total  (   -0.0004,    0.0000,   -0.5254 )
ionic       (   -0.0000,    0.0000,   -0.5889 )
crystal     (    0.0120,    0.0058,    0.1788 )


== Crystal Dipole ==

mu   =  (    0.1736,    0.0802,    9.8587 ) au
|mu| =     9.8606 au,      25.0616 Debye


== Molecular Dipole wrt Center of Mass ==

mu   =  (    0.0051,   -0.0005,   -0.8884 ) au
|mu| =     0.8884 au,       2.2581 Debye

 output psi filename:./ch3cl.pspw.movecs


== Timing ==

cputime in seconds
  prologue    :   0.432744E+01
  main loop   :   0.225073E+01
  epilogue    :   0.649500E-02
  total       :   0.658466E+01
  cputime/step:   0.357258E-01       (      63 evalulations,      21 linesearches)


Time spent doing                        total          step       percent
  total time                 :   0.658629E+01  0.104544E+00       100.0 %
  i/o time                   :   0.157849E+01  0.250553E-01        24.0 %
  FFTs                       :   0.767669E+00  0.121852E-01        11.7 %
  dot products               :   0.972802E-01  0.154413E-02         1.5 %
  geodesic                   :   0.238213E+00  0.378116E-02         3.6 %
  ffm_dgemm                  :   0.278628E-01  0.442267E-03         0.4 %
  fmf_dgemm                  :   0.227967E+00  0.361852E-02         3.5 %
  m_diagonalize              :   0.107096E-02  0.169994E-04         0.0 %
  exchange correlation       :   0.101739E+01  0.161490E-01        15.4 %
  local pseudopotentials     :   0.915766E-03  0.145360E-04         0.0 %
  non-local pseudopotentials :   0.198746E+00  0.315470E-02         3.0 %
  hartree potentials         :   0.540778E-02  0.858377E-04         0.1 %
  ion-ion interaction        :   0.139403E-02  0.221275E-04         0.0 %
  structure factors          :   0.468995E+00  0.744437E-02         7.1 %
  phase factors              :   0.200262E-04  0.317876E-06         0.0 %
  masking and packing        :   0.256437E+00  0.407043E-02         3.9 %
  queue fft                  :   0.626588E+00  0.994584E-02         9.5 %
  queue fft (serial)         :   0.383699E+00  0.609046E-02         5.8 %
  queue fft (message passing):   0.221666E+00  0.351851E-02         3.4 %

     >>>  JOB COMPLETED     AT Wed Sep 30 13:18:12 2015  <<<

 Task  times  cpu:        6.6s     wall:        6.6s


                                NWChem Input Module
                                -------------------


 >>>> PSPW Parallel Module - steepest_descent <<<<
          ****************************************************
          *                                                  *
          *     Car-Parrinello microcluster calculation      *
          *                                                  *
          *     [     steepest descent minimization   ]      *
          *                                                  *
          *     [ NorthWest Chemistry implementation ]       *
          *                                                  *
          *            version #5.00   06/01/99              *
          *                                                  *
          *    This code was developed by Eric J. Bylaska,   *
          *    and was based upon algorithms and code        *
          *    developed by the group of Prof. John H. Weare *
          *                                                  *
          ****************************************************
     >>>  JOB STARTED       AT Wed Sep 30 13:18:12 2015  <<<
          ================ PSPW input data ===================

 input psi filename:./ch3cl.pspw.movecs

 number of processors used:         1
 processor grid           :   1 x   1
 parallel mapping         :   2d hilbert
 parallel mapping         :     balanced
 parallel io              :       off
 number of threads        :         1

 options:
      ionic motion         = no
      boundary conditions  = periodic    (version3)
      electron spin        = restricted
      exchange-correlation = PBE96 (White and Bird) parameterization

 elements involved in the cluster:
      1: C   valence charge:  4.0000  lmax=  2
             comment    : Parameterized (J.Phys.Chem., vol 100, page 6966) Hamman psp
             pseudpotential type            :   0
             highest angular component      :   2
             local potential used           :   2
             number of non-local projections:   4
             cutoff =   0.800   0.850   0.850

      2: Cl  valence charge:  7.0000  lmax=  2
             comment    : Troullier-Martins pseudopotential
             pseudpotential type            :   0
             highest angular component      :   2
             local potential used           :   2
             number of non-local projections:   4
             semicore corrections included  :  1.300 (radius)  0.269 (charge)
             cutoff =   1.340   1.450   1.450

      3: H   valence charge:  1.0000  lmax=  1
             comment    : Parameterized (Chem.Phys.Lett., vol 322, page 447) Hamman psp
             pseudpotential type            :   0
             highest angular component      :   1
             local potential used           :   1
             number of non-local projections:   1
             cutoff =   0.800   0.800


 total charge=   0.000

 atomic composition:
     C :    1     Cl:    1     H :    3

 initial position of ions:
        1 C    (   -0.00000    0.00000   -2.15669 ) - atomic mass=  12.000          
        2 Cl   (   -0.00000    0.00000    1.25580 ) - atomic mass=  34.969          
        3 H    (   -1.11183   -1.60769   -2.80282 ) - atomic mass=   1.008          
        4 H    (   -0.83638    1.76672   -2.80282 ) - atomic mass=   1.008          
        5 H    (    1.94822   -0.15903   -2.80282 ) - atomic mass=   1.008          
        G.C.   (   -0.00000    0.00000   -1.86187 )
        C.O.M. (   -0.00000    0.00000    0.19122 )
        number of constraints =      0 ( DOF =      9 )

 number of electrons: spin up=     7 (   7 per task)  down=     7 (   7 per task) (Fourier space)
 number of orbitals : spin up=     7 (   7 per task)  down=     7 (   7 per task) (Fourier space)

 supercell:
      lattice:    a1=<  24.000   0.000   0.000 >
                  a2=<   0.000  24.000   0.000 >
                  a3=<   0.000   0.000  24.000 >
      reciprocal: b1=<   0.262   0.000   0.000 >
                  b2=<   0.000   0.262   0.000 >
                  b3=<   0.000   0.000   0.262 >
      volume :      13824.0
      density cutoff=  8.773  fft= 32x 32x 32(     8536 waves     8536 per task)
      wavefnc cutoff=  8.773  fft= 32x 32x 32(     8536 waves     8536 per task)
      Ewald summation: cut radius=    7.64  and  1
                        madelung= 1.76011888

 technical parameters:
      translation constrained
      time step=     15.80     ficticious mass=  400000.0
      tolerance=.100E-08 (energy)   0.100E-08 (electron)   0.100E-03 (ion)
      maximum iterations =       100 (   10 inner     10 outer )



          ================ iteration =========================
     >>>  ITERATION STARTED AT Wed Sep 30 13:18:12 2015  <<<
    iter.           Energy         DeltaE       DeltaRho 
    ------------------------------------------------------
      10   -0.2256232186E+02   -0.14280E+00    0.11735E-09    0.00000E+00
      20   -0.2256232187E+02   -0.93469E-10    0.71300E-10    0.00000E+00
  *** tolerance ok.     iteration terminated.
     >>>  ITERATION ENDED   AT Wed Sep 30 13:18:12 2015  <<<



          =============  summary of results  =================
 final position of ions:
        1 C    (   -0.00000    0.00000   -2.15669 ) - atomic mass=  12.000          
        2 Cl   (   -0.00000    0.00000    1.25580 ) - atomic mass=  34.969          
        3 H    (   -1.11183   -1.60769   -2.80282 ) - atomic mass=   1.008          
        4 H    (   -0.83638    1.76672   -2.80282 ) - atomic mass=   1.008          
        5 H    (    1.94822   -0.15903   -2.80282 ) - atomic mass=   1.008          
        G.C.   (   -0.00000    0.00000   -1.86187 )
        C.O.M. (   -0.00000    0.00000    0.19122 )
        number of constraints =      0 ( DOF =      9 )

 number of electrons: spin up=    7.00000  down=    7.00000 (real space)


 total     energy    :  -0.2256232187E+02 (   -0.45125E+01/ion)
 total orbital energy:  -0.6225340952E+01 (   -0.88933E+00/electron)
 hartree   energy    :   0.2602384337E+02 (    0.37177E+01/electron)
 exc-corr  energy    :  -0.5871079731E+01 (   -0.83873E+00/electron)
 ion-ion   energy    :   0.8125323521E+01 (    0.16251E+01/ion)

 K.S. kinetic energy :   0.1088218622E+02 (    0.15546E+01/electron)
 K.S. V_l  energy    :  -0.6516635024E+02 (   -0.93095E+01/electron)
 K.S. V_nl energy    :   0.3443754989E+01 (    0.49196E+00/electron)
 K.S. V_Hart energy  :   0.5204768673E+02 (    0.74354E+01/electron)
 K.S. V_xc energy    :  -0.7432618655E+01 (   -0.10618E+01/electron)
 Virial Coefficient  :  -0.1572067122E+01

 orbital energies:
    -0.2507699E+00 (  -6.824eV)
    -0.2508115E+00 (  -6.825eV)
    -0.3900594E+00 ( -10.614eV)
    -0.4039666E+00 ( -10.993eV)
    -0.4047822E+00 ( -11.015eV)
    -0.6321298E+00 ( -17.201eV)
    -0.7801511E+00 ( -21.229eV)

 Total PSPW energy   :  -0.2256232187E+02


=== Spin Contamination ===

 <Sexact^2> =    0.0000000000000000     
 <S^2>      =    0.0000000000000000     



== Center of Charge ==

spin up     (   -0.0004,    0.0000,   -0.5254 )
spin down   (   -0.0004,    0.0000,   -0.5254 )
     total  (   -0.0004,    0.0000,   -0.5254 )
ionic       (   -0.0000,    0.0000,   -0.5889 )
crystal     (    0.0120,    0.0058,    0.1788 )


== Crystal Dipole ==

mu   =  (    0.1736,    0.0802,    9.8587 ) au
|mu| =     9.8606 au,      25.0616 Debye


== Molecular Dipole wrt Center of Mass ==

mu   =  (    0.0051,   -0.0005,   -0.8884 ) au
|mu| =     0.8884 au,       2.2581 Debye

 output psi filename:./ch3cl.pspw.movecs

 -----------------
 cputime in seconds
 prologue    :   0.10470509429113761     
 main loop   :   0.64731383323669434     
 epilogue    :    2.5056123733520508E-002
 total       :   0.77707505126135246     
 cputime/step:    3.2365691661834714E-002



Time spent doing                        total          step       percent
  total time                 :   0.777112E+00  0.388556E-01       100.0 %
  i/o time                   :   0.213349E-01  0.106674E-02         2.7 %
  FFTs                       :   0.150053E+00  0.750265E-02        19.3 %
  dot products               :   0.301938E-01  0.150969E-02         3.9 %
  geodesic                   :   0.242472E-02  0.121236E-03         0.3 %
  two-electron Gaussian      :   0.199367E+00  0.996834E-02        25.7 %
  ffm_dgemm                  :   0.158422E-01  0.792110E-03         2.0 %
  fmf_dgemm                  :   0.123606E-01  0.618029E-03         1.6 %
  m_diagonalize              :   0.591278E-04  0.295639E-05         0.0 %
  exchange correlation       :   0.275998E+00  0.137999E-01        35.5 %
  local pseudopotentials     :   0.635575E-02  0.317788E-03         0.8 %
  non-local pseudopotentials :   0.601430E-01  0.300715E-02         7.7 %
  hartree potentials         :   0.733130E-03  0.366565E-04         0.1 %
  ion-ion interaction        :   0.694990E-03  0.347495E-04         0.1 %
  structure factors          :   0.889818E-02  0.444909E-03         1.1 %
  phase factors              :   0.812877E-04  0.406439E-05         0.0 %
  masking and packing        :   0.273961E-01  0.136981E-02         3.5 %
  queue fft                  :   0.198103E+00  0.990517E-02        25.5 %
  queue fft (serial)         :   0.120562E+00  0.602811E-02        15.5 %
  queue fft (message passing):   0.708134E-01  0.354067E-02         9.1 %
  MATHIAS InnerLoop          :   0.647193E+00  0.323597E-01        83.3 %
  MATHIAS Phaze              :   0.149984E-03  0.749921E-05         0.0 %
  MATHIAS Pipelined FFTs     :   0.101715E+00  0.508577E-02        13.1 %
  MATHIAS Lagrange           :   0.328715E-01  0.164357E-02         4.2 %
  MATHIAS Exch Corr          :   0.276045E+00  0.138022E-01        35.5 %
  MATHIAS Hpsi               :   0.193549E+00  0.967746E-02        24.9 %
     >>>  JOB COMPLETED     AT Wed Sep 30 13:18:12 2015  <<<

 Task  times  cpu:        0.8s     wall:        0.8s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:    8        8       97       10        0        0        0        0     
number of processes/call 1.00e+00 1.00e+00 0.00e+00 0.00e+00 0.00e+00
bytes total:             2.61e+04 9.68e+03 0.00e+00 0.00e+00 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 7744 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	       273	        24
	current total bytes		         0	         0
	maximum total bytes		  29758744	   5986792
	maximum total K-bytes		     29759	      5987
	maximum total M-bytes		        30	         6


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
        S. Fischer, S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen,
    V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus, M. Malagoli,
       A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati,
     J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, M. Williamson, R. J. Harrison,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
    M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
      G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall,
      J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt,
      P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall,
    D. Elwood, E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson,
     J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        7.4s     wall:        7.4s
